pkg/apis/config/feature_flags.go

```diff
@@ -142,7 +142,7 @@ var (
 	DefaultEnableKeepPodOnCancel = PerFeatureFlag{
 		Name:      KeepPodOnCancel,
 		Stability: AlphaAPIFields,
-		Enabled:   DefaultAlphaFeatureEnabled,
+		Enabled:   DefaultBetaFeatureEnabled,
```

Comment:
Should this PR be called `promote keep-pod-on-cancel to beta` instead? I think that would reflect what is happening in the PR and the effect of the PR better. The title currently only reflects the effect of the PR changing the name to the one I mentioned would give a more holistic explanation of the PR. Also, won't this feature only be enabled by default if the feature is stable ?

Reply:
cannot see the resolved button here so manually noting here that this has been resolved by https://github.com/tektoncd/pipeline/pull/8343#issuecomment-2763019598